schema.yaml

```diff
@@ -616,3 +622,16 @@ schema;reference:
     year-original:
       required: False
       type: int
+
+schema;identifier:
+  type: map
+  mapping:
+
+    type:
+      required: True
+      type: str
+      enum: ['software-heritage', 'other']
```

Comment:
I can suggest a longer enum of identifier types with:
```
['ASCL-ID', 'HAL-ID', 'SwMath-ID', 'Wikidata entity', 'FSF directory ID']
```
or package identifier would depend on the package manager:
```
['NPM', 'PyPI', 'Maven']
```